packages/migration-graph-shared/src/entities/package.ts

```diff
@@ -2,9 +2,13 @@
 import path, { resolve } from 'path';
 import { readJsonSync, writeJsonSync } from 'fs-extra';
 import sortPackageJson from 'sort-package-json';
+import { sync as fastGlobSync } from 'fast-glob';
 
 import { removeNestedPropertyValue, setNestedPropertyValue } from '../utils/pojo';
 import { getWorkspaceGlobs } from '../utils/workspace';
+import { Graph } from '../graph';
```

Comment:
Looks like Graph and ModuleNode are types?

```suggestion
import type { Graph } from '../graph';
import type { ModuleNode } from '../types';
```